src/helper/constants.js

```diff
@@ -1,3 +1,4 @@
-const CATEGORIES = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+const CATEGORIES = ['Select Category', 'Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+const CATEGORIES_FILTER = ['All', 'Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
 
-export default CATEGORIES;
+export { CATEGORIES, CATEGORIES_FILTER };
```

Comment:
**OPTIONAL**: The two arrays are almost identical... remember DRY. Why not have one array and expand that one with the values initial values:

```js
const CATEGORIES = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];

// somewhere
['Select Category', ...CATEGORIES]

// somewhere
['All', ...CATEGORIES]
```